compiler/src/iree/compiler/Codegen/Passes.cpp

```diff
@@ -39,19 +39,19 @@ void registerCodegenPasses() {
         buildLLVMCPUCodegenPassPipeline(passManager);
       });
 
-  static PassPipelineRegistration<> LinalgNVVMPipeline(
-      "iree-codegen-linalg-to-nvvm-pipeline",
-      "Runs the progressive lowering pipeline from Linalg to NVVM",
-      [](OpPassManager &passManager) {
-        buildLLVMGPUTransformPassPipeline(passManager, false);
-      });
+  // static PassPipelineRegistration<> LinalgNVVMPipeline(
+  //     "iree-codegen-linalg-to-nvvm-pipeline",
+  //     "Runs the progressive lowering pipeline from Linalg to NVVM",
+  //     [](OpPassManager &passManager) {
+  //       buildLLVMGPUTransformPassPipeline(passManager, false);
+  //     });
 
-  static PassPipelineRegistration<> LinalgROCDLPipeline(
-      "iree-codegen-linalg-to-rocdl-pipeline",
-      "Runs the progressive lowering pipeline from Linalg to ROCDL",
-      [](OpPassManager &passManager) {
-        buildLLVMGPUTransformPassPipeline(passManager, true);
-      });
+  // static PassPipelineRegistration<> LinalgROCDLPipeline(
+  //     "iree-codegen-linalg-to-rocdl-pipeline",
+  //     "Runs the progressive lowering pipeline from Linalg to ROCDL",
+  //     [](OpPassManager &passManager) {
+  //       buildLLVMGPUTransformPassPipeline(passManager, true);
+  //     });
```

Comment:
What should be done about these passes? Would there be a way to register these inside of each Target?

Reply:
are there any tests with these? We could run just the `-iree-hal-transformation-pipeline` (maybe) and set CUDA/RoCM backends there